tests/static/js/jqunit/jqunit-binder-array.js

```diff
@@ -0,0 +1,45 @@
+/* globals fluid */
```

Comment:
These are "static" as opposed to "generated using a template".  Alternate names are welcome.